bucket/cnping.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "1.0.0",
+    "description": "Minimal Graphical IPV4 Ping/HTTP Ping Tool.",
+    "homepage": "https://github.com/cntools/cnping",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/cntools/cnping/releases/download/1.0.0/cnping.exe",
+            "hash": "ed61944a9d720fd0ad429fa7ec97c2357fa805301532dd43049ffc5666bfdc8e"
+        }
+    },
+    "bin": "cnping.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/cntools/cnping/releases/download/$version/cnping.exe",
+                "hash": "ed61944a9d720fd0ad429fa7ec97c2357fa805301532dd43049ffc5666bfdc8e"
+            }
+        }
```

Comment:
```suggestion
        "architecture": {
            "64bit": {
                "url": "https://github.com/cntools/cnping/releases/download/$version/cnping.exe"
            }
        },
        "hash": {
            "url": "https://github.com/cntools/cnping/releases/tag/$version",
            "find": "SHA256:\\s+$sha256\\s+$basename"
        }
```

Reply:
This hash regex rely on human-written release description. For example, for beta versions there is no hash in description. I don't think this is more reliable than just specifing hash in manifest.

Reply:
> This hash regex rely on human-written release description. For example, for beta versions there is no hash in description. I don't think this is more reliable than just specifing hash in manifest.

Hi @ivanik7.

The idea is that [autoupdate](https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate) section is used to update manifest automatically (when `checkver` detects new stable version). So we shouldn't use any hard-coded values here: either we can detect hash automatically (from release notes, file, etc), or we exclude autoupdate.hash from manifest completely and hash will be calculated for downloaded binary.